api/employee/employeeModel.js

```diff
@@ -0,0 +1,43 @@
+const db = require('../../data/dbConfig');
```

Comment:
I think we'll want a model that gets all employees by org id

Reply:
This is all set.